snuba/cli/rust_consumer.py

```diff
@@ -153,11 +148,20 @@ class ClickhouseClusterConfig:
     database: str
 
 
+@dataclass(frozen=True)
+class MessageProcessorConfig:
+    # TODO: Probably won't end up needing all this
+    python_class_name: str
+    python_module: str
+    python_file_path: str
```

Comment:
Does this mean the whole processor needs to be in one file ?

Reply:
we only need `name` and args from the storage yaml, the `DatasetMessageProcessor` from the baseclass can load by classname: https://github.com/getsentry/snuba/blob/7e8df53a16171dcd3ff06f610ef1d0aae8b84006/rust_snuba/src/strategies/python.rs#L33-L35

alternatively we can also pass the full classpath `snuba.processors....Processor` so we don't rely on this class registry in Python.

`file_path` I can't use for anything.

We are not constrained to a single file... we launch a real Python interpreter and can import the entirety of Snuba

Reply:
I see. Cool

Reply:
Removed the file path